### PR TITLE
Added completed pull request template from team C

### PR DESCRIPTION
**Pull request by jmmiddour**
### Location:

- pull_request_template.md

---

### Details:

1. Team C worked together to complete our pull request template

---

### Committed Features:

1. Added completed pull request template from team C

---

### Any Issues:

No issues

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+[Link to the PR Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)
+
+---
+
+( BELOW IS AN EXAMPLE )
+
+---
+
```
Comment (thomasj-s) [inline pull_request_template.md:12]: I like having the link to the trello card here.

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+[Related Trello Card](your link here)
+
+---
+
```
Comment (thomasj-s) [inline pull_request_template.md:16]: A little confused about what goes in the 'location' section

**Review by thomasj-s — APPROVED** [review]
Lean, but has everything we need.  Other than not fully understanding the 'location' section.

File: pull_request_template.md (hunk 2 above)
Comment (jmmiddour) [reply pull_request_template.md:16]: The 'Location' section is where you put the files that you touched/worked on in which you are doing the pull request for. Does that help to clarify things?

**Review by jeremy-jewett — APPROVED** [review]
It looks great.

**Pull request merged**